app/controllers/Message.scala

```diff
@@ -85,6 +108,9 @@ object Message extends LilaController {
 
   def delete(id: String) = AuthBody { implicit ctx =>
     implicit me =>
-      api.deleteThread(id, me) inject Redirect(routes.Message.inbox(1))
+      negotiate (
+        html = api.deleteThread(id, me) inject Redirect(routes.Message.inbox(1)),
+        api = _ => api.deleteThread(id, me) inject Ok(Json.obj("ok" -> true, "id" -> id))
```

Comment:
I thought it might be useful to someone someday, but the more I think about it, the harder it is to conceive of a use case. Deleted.